bot/__init__.py

```diff
@@ -233,6 +233,14 @@ async def on_message(self, message):
                     )
                 )
 
+        # filter social invites from unverified members
+        if len(message.author.roles) == 0:
+            for link in [x[0] for x in re.findall(URL_REGEX, message.content)] 
+                if any(domain in link.lower() for domain in BLOCKED_DOMAINS):
+                    await message.delete()
+                    await message.author.ban()
+                    break
```

Comment:
Can we also send a message to the #bot channel for logging purposes, in case some mistake happens